### PR TITLE
[12.0][MIG] stock_orderpoint_uom

### PR DESCRIPTION
**Pull request by kittiu**
Migrate to v12

### CODE REVIEW

File: stock_orderpoint_uom/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Orderpoint UoM",
+    "summary": "Allows to create procurement orders in the UoM indicated in "
+               "the orderpoint",
+    "version": "11.0.1.0.0",
```
Comment (rousseldenis) [inline stock_orderpoint_uom/__manifest__.py]: @kittiu Change to '12.0.1.0.0'. This can relaunch tests correctly

Comment (kittiu) [reply stock_orderpoint_uom/__manifest__.py]: @rousseldenis thank you and fixed.

**Comment by LoisRForgeFlow** [comment]
You lost commit history here, could you put it back?

Thanks

**Comment by kittiu** [comment]
@lreficent sorry for the mistake. I am not sure how to bring it back other than do it again, this is correct way?
https://github.com/OCA/stock-logistics-warehouse/pull/568
(please close this one)

**Comment by LoisRForgeFlow** [comment]
You can always force push to the branch of an open PR, in order to avoid to open a new one.

**Pull request closed without merging**